### DISCUSSION TITLE
Entire screen?

### DISCUSSION THREAD

**Discussion by danieltty** [rxhanson/Rectangle#1510] [Q&A]
Not able to switch to entire screen (max to entire screen)?

**Comment by rxhanson** [comment]
Are you looking for a keyboard shortcut or snap area to make a window "fullscreen"? The keyboard shortcut for that is `control+command+f`. There's no snap area option for this in Rectangle, although there is one in [Rectangle Pro](https://rectangleapp.com/pro) (paid).
